source/common/http/utility.cc

```diff
@@ -80,6 +80,9 @@ std::string Utility::parseCookieValue(const HeaderMap& headers, const std::strin
       for (const std::string& s : StringUtil::split(std::string{header.value().c_str()}, ';')) {
         size_t first_non_space = s.find_first_not_of(" ");
         size_t equals_index = s.find('=');
+        if (first_non_space == std::string::npos || equals_index == std::string::npos) {
+            break;
```

Comment:
Ahh, I think it should be continue. Sure, I'll try to add some comments. I'd happily restructure to remove the break/continue too if you'd prefer. I am not sure if a guard was preferred style in this case.